packages/design-system-react-native/src/temp-components/Spinner/Spinner.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { render } from '@testing-library/react-native';
+import React from 'react';
+
+import Spinner from './Spinner';
+
+describe('Spinner', () => {
+  it('renders the spinner icon', () => {
+    const { getByTestId } = render(<Spinner />);
+    const iconElement = getByTestId('spinner-icon');
+
+    expect(iconElement).toBeDefined();
+  });
+
+  it('renders the loading text when provided', () => {
+    const loadingText = 'Loading...';
+    const { getByText } = render(<Spinner loadingText={loadingText} />);
+
+    expect(getByText(loadingText)).toBeDefined();
+  });
+
+  it('renders correctly without loading text', () => {
+    const { queryByText } = render(<Spinner />);
+
+    expect(queryByText('Loading...')).toBeNull();
+  });
```

Comment:
non-blocking: Could you add `expect(queryByText('Loading...')).toBeNull();` to the first test to consolidate and reduce the number of tests?